content/docs/reference-react.md

```diff
@@ -227,4 +228,9 @@ You can also use it with the shorthand `<></>` syntax. For more information, see
 `React.forwardRef` accepts a render function that receives `props` and `ref` parameters and returns a React node. Ref forwarding is a technique for passing a [ref](/docs/refs-and-the-dom.html) through a component to one of its descendants. This technique can be particularly useful with [higher-order components](/docs/higher-order-components.html):
 `embed:reference-react-forward-ref.js`
 
-For more information, see [forwarding refs](/docs/forwarding-refs.html).
\ No newline at end of file
+For more information, see [forwarding refs](/docs/forwarding-refs.html).
+
+### `React.createRef`
+
+`React.createRef` creates a ref that can be attached to React elements via the ref attribute.
+`embed:16-3-release-blog-post/create-ref-example.js`
```

Comment:
Maybe link to "refs and the DOM" here? It's not clear what this means to someone who's seeing refs for the first time.

Reply:
I've linked the word `ref` to the "refs and the DOM" section.